### PR TITLE
#1124 wp revamp changes tab

### LINKED ISSUE #1124: Work Package Detail Revamp - Changes Tab
Blocked by #1121 

Populate the Changes tab with content (should have the same data as the changes section did on the old wp detail page, just without the pageblocks, see the project detail page or this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1112, for an example of how to do that):

This PR should merge into a feature branch called "#1097-wp-detail-revamp" instead of develop.

image

Epic: #1097

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #1124  (issue #)

### CODE REVIEW

**Comment by LOTaher** [comment]
## Changes

Added content to the changes tab by changing `ChangesList.tsx & removed the pageblock component from `BulletList.tsx`

## Notes

The BulletList changes should not interfere anywhere else in the UI (?)

## Screenshots

image

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1124  (issue #1124)

File: src/frontend/src/components/ChangesList.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const ChangesList: React.FC<ChangesListProps> = ({ changes }) => {
           <Link component={RouterLink} to={`${routes.CHANGE_REQUESTS}/${ic.changeRequestId}`}>
             #{ic.changeRequestId}
           </Link>
-          ]
+          ]{' '}
```
Comment (RChandler234) [inline src/frontend/src/components/ChangesList.tsx:29]: this looks like this on my local rn 
image
I would look at what I did on the changes tab of the projects page, copy that into the changes component and use it for both

**Review by RChandler234 — COMMENTED** [review]
Also DescriptionList.tsx and its associated test file are unused now, can you delete them
